commit af4f77cb8baf801794e96c0b74d495d8289cd128
Author: Christian Brauner <christian.brauner@ubuntu.com>
Date:   Sun Jul 2 23:32:13 2017 +0200

    import: remove last dependency on symlink
    
    When importing containers we do not rely on symlinks.
    
    Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

diff --git a/lxd/api_internal.go b/lxd/api_internal.go
index 3a520be..a41746f 100644
--- a/lxd/api_internal.go
+++ b/lxd/api_internal.go
@@ -182,7 +182,8 @@ func internalImport(d *Daemon, r *http.Request) Response {
 	}
 
 	// Read in the backup.yaml file.
-	backup, err := slurpBackupFile(shared.VarPath("containers", req.Name, "backup.yaml"))
+	backupYamlPath := shared.VarPath("storage-pools", containerPoolName, "containers", req.Name, "backup.yaml")
+	backup, err := slurpBackupFile(backupYamlPath)
 	if err != nil {
 		return SmartError(err)
 	}
